src/PlaywrightSharp.Chromium/Messaging/Browser/BrowserCloseRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace PlaywrightSharp.Chromium.Messaging.Browser

```

Comment:
we should consider generating the protocol classes in a similar way to playwright - [utils\protocol-types-generator\index.js](https://github.com/kblok/playwright/blob/v0.10.0/utils/protocol-types-generator/index.js)

Reply:
Issue created! https://github.com/kblok/playwright-sharp/issues/229

Feel free to grab it. I know you like those tools :)